src/redux/modules/notifications.js

```diff
@@ -0,0 +1,49 @@
+// action types
+export const CREATE_NOTIFICATION = 'whatever/notifications/CREATE_NOTIFICATION';
+const DELETE_NOTIFICATION = 'whatever/notifications/DELETE_NOTIFICATION';
+
+// initial state
+const initialState = {
+  items: [],
+};
+
+// action creators
+export const createNotification = notification => dispatch => {
+  window.setTimeout(
```

Comment:
`window.setTimeout` returns an ID which you can use to get the instance of that timeout. Maybe it could be useful to store that reference to state too, since then you'll be able to cancel the timeout.